galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/cicsresource/IJvmserverLog.java

```diff
@@ -143,46 +144,55 @@ public interface IJvmserverLog {
 	/**
 	 * Searches contents of log for specified search or fail Pattern since the last checkpoint
 	 * @param searchPattern the Pattern to search
+	 * @param failPattern the failure pattern to search
 	 * @return the string found or null
 	 * @throws CicsJvmserverResourceException 
 	 */
 	public String searchForPatternSinceCheckpoint(Pattern searchPattern, Pattern failPattern) throws CicsJvmserverResourceException;
 
 	/**
-	 * Wait for a search search String to appear in specified log. Will check every 3 seconds until one of:
+	 * Wait for a search search String to appear in specified log. 

```

Comment:
should it be just search String rather than search search String?

Reply:
Agreed.

Reply:
Fix ready to push up shortly.